### ISSUE TITLE
Hist:divide is broken

### ISSUE THREAD

**Issue by kratsg** [rootpy/rootpy#659]
```
An exception was caught!
Traceback (most recent call last):
  File "totw.py", line 363, in <module>
    ratio = Hist.divide(hist, hstack)
  File "/Users/kratsg/.virtualenvs/TakeOverTheWorld/lib/python2.7/site-packages/rootpy/plotting/hist.py", line 454, in divide
    ROOT.TH1.Divide(ratio, h1, h2, c1, c2, option)
TypeError: none of the 3 overloaded methods succeeded. Full details:
  Bool_t TH1::Divide(const TH1* h1) =>
    takes at most 1 arguments (5 given)
  Bool_t TH1::Divide(TF1* f1, Double_t c1 = 1) =>
    takes at most 2 arguments (5 given)
  Bool_t TH1::Divide(const TH1* h1, const TH1* h2, Double_t c1 = 1, Double_t c2 = 1, Option_t* option = "") =>
    could not convert argument 2
```

This is defined here: https://github.com/rootpy/rootpy/blob/master/rootpy/plotting/hist.py#L451-L459

```
ROOT 5.34/23 (heads/v5-34-00-patches@v5-34-22-106-g4a0dea3, Nov 19 2014, 08:46:00 on macosx64)
```

and

```
rootpy (0.8.0)
```

**Comment by kratsg** [comment]
Ahh, it looks like it doesn't like `THStack` objects for any of the arguments unfortunately. Could potentially avoid the error by doing something like

```
if hasattr(hist, 'GetHistogram'):
  hist = hist.GetHistogram()
```

in the code.

**Comment by ndawe** [comment]
Yes, thanks! I'll add that.

**Comment by ndawe** [comment]
Fixed in #705
